src/lib/currency/currency.ml

```diff
@@ -195,6 +195,91 @@ end = struct
     let%bind actual = range_check' t in
```

Comment:
delete?

Reply:
We still use it for `<=` and friends, sadly.